src/bot/commands/help_command_info.rs

```diff
@@ -97,7 +98,9 @@ impl HelpCommandInfo {
             HelpCommand::Help => HELP.to_string(),
             HelpCommand::Subscribe => SUBSCRIBE.to_string(),
             HelpCommand::Unsubscribe => UNSUBSCRIBE.to_string(),
-            HelpCommand::ListSubscriptions => LIST_SUBSCRIPTIONS.to_string(),
+            HelpCommand::ListSubscriptions => {
```

Comment:
help_command_info is used for showing information. so it shouldn't show actual subscriptions